Sources/Core/Sources/Game.swift

```diff
@@ -267,8 +267,16 @@ public struct Game {
   /// Gets the position of the block currently targeted by the player.
   public func targetedBlock() -> BlockPosition? {
     var ray: Ray = Ray(origin: .zero, direction: .zero)
+    var targetable: Bool = true
```

Comment:
I think the gamemode check should be in the renderer instead of the `targetedBlock` method because in spectator I would still expect a targeted block to be returned (for f3 screen purposes), it just wouldn't have an outline rendered

Reply:
Fair enough. I haven't seen any usage of this function outside of `worldRenderer` which is why i went with this, a little bit destructive, approach. Corrected.